CONTRIBUTING.md

```diff
@@ -2,8 +2,26 @@
 
 The team behind the [OSS Review Toolkit](https://github.com/heremaps/oss-review-toolkit) gratefully accepts contributions via
 [pull requests](https://help.github.com/articles/about-pull-requests/) filed against the
-[GitHub project](https://github.com/heremaps/oss-review-toolkit/pulls). As part of filing a pull request we ask you to
-sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
+[GitHub project](https://github.com/heremaps/oss-review-toolkit/pulls).
+
+# Signing each Commit
+As part of filing a pull request we ask you to
+sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO) in each commit.
```

Comment:
"sign off the" should still do to the previous line.

Reply:
Done.

Reply:
It does not seem to be done 😉